### PR TITLE
76 add diagrams

### PR DESCRIPTION
**Pull request by gem09lo**
took images out of gitignores

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,25 +15,24 @@ The primary objective is to build a scalable and efficient cloud-hosted data pip
...
 **Visitors**
-![alt text](image-4.png)
-
+![alt text](images/image-4.png)
 
 **Staff**
-![alt text](image-3.png)
-![alt text](image-1.png)
+![alt text](images/image-3.png)
+![alt text](images/image-1.png)
```
Comment (jk-george) [inline README.md:36]: Love it, looks very nice!

**Review by jk-george — APPROVED** [review]
Merge to add images into the README: Approved

**Review by jk-george — APPROVED** [review]
Merges to solve conflict on gitignore

**Review by emily-curtis — APPROVED** [review]
🤠

**Pull request merged**